source/extensions/filters/http/cache/cache_headers_utils.cc

```diff
@@ -168,6 +168,102 @@ absl::optional<uint64_t> CacheHeadersUtils::readAndRemoveLeadingDigits(absl::str
   return absl::nullopt;
 }
 
+absl::flat_hash_set<std::string> VaryHeader::parseAllowlist() {
+  // TODO(cbdm): Populate the hash_set from
+  // envoy::extensions::filters::http::cache::v3alpha::CacheConfig::allowed_vary_headers.
+  // Need to make sure that the headers we add here are valid values (i.e., not malformed). That
+  // way, we won't have to check this again in isAllowed.
+  return {"accept"};
```

Comment:
It might be best to name this something like `x-temporary-standin-header-name` to make it extra-clear that this is just a temporary crutch.